server/routes/makeAReferral/risk-information/oasys/oasysRiskSummaryView.ts

```diff
@@ -103,8 +103,8 @@ export default class OasysRiskSummaryView {
         },
       },
       additionalRiskInformation: {
-        label: this.supplementaryRiskInformation ? undefined : this.noAdditionalRiskInformationLabel,
-        text: this.supplementaryRiskInformation ? this.supplementaryRiskInformation.riskSummaryComments : null,
+        label: this.noAdditionalRiskInformationLabel,
+        text: null,
```

Comment:
Could we remove the parameter `supplementaryRiskInformation` altogether, if we're not using it?

Reply:
```
server/routes/shared/showReferralView.ts
16:    this.supplementaryRiskInformationView = new OasysRiskSummaryView(presenter.riskInformation, presenter.riskSummary)
```
seems to be still passing it through but it won't have any effect it seems

Reply:
Yes, you're right. I've improved on this by splitting the functionality into two separate classes:

OAsysRiskInformation and ArnRiskInformation since they're effectively two different things.